ivy/functional/backends/torch/experimental/losses.py

```diff
@@ -102,18 +103,8 @@ def soft_margin_loss(
     )
 
 
-@with_unsupported_dtypes(
-    {
-        "2.0.1 and below": (
-            "float16",
-            "uint8",
-            "int8",
-            "int16",
-            "int32",
-            "int64",
-            "bool",
-        )
-    },
+@with_supported_dtypes(
+    {"2.0.1 and below": "float"},
```

Comment:
The error message in [this](https://github.com/unifyai/ivy/actions/runs/6301635175/job/17107481998?pr=26000#step:3:30698) test failure is coming from here, basically it should be `("float",)` instead of `"float"`

Reply:
updated it, but it looks like the tests are not running now due to `dill` not being available. Thanks!